gems/policy-parser/CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Unreleased
```

Comment:
Seems like we should probably use the main project's `CHANGELOG` rather than maintaining a separate one from now on. If so, maybe put a comment to this effect at the top of this file so that it's clear which one is active?